### ISSUE TITLE
[USABILITY]

### ISSUE THREAD

**Issue by joegrohens** [rokwire/safer-illinois-app#427]
**Usability issue**
The return of the one-time code for non-university members attributes the message to "Illinois App" instead of "Safer Illinois".

**To Reproduce**
Get Started > Select Non University Member > Verify Phone Number > Enter your cell phone number

The system returned this message:

> +1 (209) 433-6287
> Your Illinois App Verification code is: 879090
(see attached screenshot)

This message refers to the wrong app.

**Solution:**
The message could be changed to:
"Your Safer Illinois verification code is: ____"
(or, for greater generalizability)
"Your verification code is: _____"

(For consistent capitalization, lower-case "verification")

**Screenshots**
![return of one-time code](https://user-images.githubusercontent.com/66988329/100138387-944a6980-2e53-11eb-84ec-4aa847a7e229.png)

**Smartphone**

Device: [iphone 11]
OS: [os 14.2]

**Comment by pmarkhennessy** [comment]
@joegrohens We have no control over this message. It comes from the Illinois backend.

@sandeep-ps

**Comment by sandeep-ps** [comment]
Yes, I can change it to "Your verification code is". Currently, both Illinois and Safer Illinois apps share the same phone verification system.

**Comment by sandeep-ps** [comment]
Hi @joegrohens, 
After little bit of digging, it looks like there is no direct way to customize the message. Twilio uses the format `Your <app name> verification code is <code>`. Currently the app name is `Illinois App`. We can change that. For example, if we change it to `app`, the message will be, `Your app verification code is <code>`, but I'm not sure if that will be particularly good.

If this is very important, we can create a new account in Twilio for Safer Illinois and most likely it would also mean deploying a new instance of the authentication building block specifically for Safer Illinois. Please let me know if this needs to be pursued. I remember this coming up when Safer Illinois app development started and it's my understanding that then we chose to go ahead with the current system.

**Comment by joegrohens** [comment]
Thanks @sandeep-ps. I don't know who also should be consulted on this. 

I would recommend deleted the <app name> variable. The message arrives within seconds, so it will not be confusing to a user. But using Illinois App in a message about Safer Illinois can contribute to confusion. 

Should we add others to this discussion? Who?

Joe

**Comment by sandeep-ps** [comment]
Hi @joegrohens, Let me add @isaac-galvan to this conversation. Isaac, what do you think about what Joe suggested? 

The first option (replacing "Illinois" with "app" in the text message) will be the quickest, but the common practice seems to be add a name to identify the app from which the text is getting sent. If there is a majority agreement, we probably just need to let JP and Bill know so that they are aware of this change.

**Comment by joegrohens** [comment]
If this call to Twilio is part of backend capabilities, I am guessing that developers of other apps would need to use it too. In which case they would not want the message to be branded w/ Illinois. 

I suppose an alternative could be to replace "Illinois" w/ "Rokwire".
